packages/widgets/src/lib/components/InferenceWidget/widgets/ConversationalWidget/ConversationalWidget.svelte

```diff
@@ -116,33 +124,29 @@
 	}
 
 	function isValidOutput(arg: any): arg is Response {
-		return (
-			arg && Array.isArray(arg?.conversation?.generated_responses) && Array.isArray(arg?.conversation?.past_user_inputs)
-		);
+		return typeof arg?.generated_text === "string";
 	}
 
-	function parseOutput(body: unknown): {
-		conversation: Conversation;
+	function parseOutput(
+		body: unknown,
+		chat: Message[]
+	): {
+		chat: Message[];
 		output: Output;
 	} {
 		if (isValidOutput(body)) {
-			const conversation = body.conversation;
-			const pastUserInputs = conversation.past_user_inputs;
-			const generatedResponses = conversation.generated_responses;
-			const output = pastUserInputs
-				.filter(
-					(x, i) =>
-						x !== null && x !== undefined && generatedResponses[i] !== null && generatedResponses[i] !== undefined
-				)
-				.map((x, i) => ({
-					input: x ?? "",
-					response: generatedResponses[i] ?? "",
-				}));
-			return { conversation, output };
+			const chatWithOutput = chat.concat([{ role: "assistant", content: body.generated_text }]);
```

Comment:
```suggestion
			const chatWithOutput = [...chat, { role: "assistant", content: body.generated_text }];
```

we use this style in our JS codebases